docs/pages/clients/getting-started.md

```diff
@@ -32,28 +32,78 @@ The Development Client uses deep links to open projects from the QR code. If you
 
 </Tabs>
 
-## Building your Development Client
+## Building and installing your first custom client
 
-You can now build your project and launch it in your iOS simulator
+### In the cloud
+
+However you choose to manage your native projects, we recommend using [EAS Build](eas-build.md) for the smoothest experience, especially if you do not have experience with Xcode and gradle builds.
+
+After you configure your project as covered by [the Building with EAS guide](eas-build.md), you can build your custom client with one command:
+
+<Tabs tabs={["For iOS Devices (Apple Developer membership required)", "For Android Devices"]}>
+
+<Tab >
+
+<InstallSection packageName="expo-dev-client" cmd={["eas build --profile development --platform ios"]} hideBareInstructions />
+
+</Tab>
+
+<Tab >
+
+<InstallSection packageName="expo-dev-client" cmd={["eas build --profile development --platform android"]} hideBareInstructions />
+
+</Tab>
+
+</Tabs>
+
+and installing the resulting build on your device.
+
+### Locally
+
+If you are comfortable setting up Xcode, Android Studio, and related dependencies, you can build and distribute your app the same as any other iOS or Android application (after running `expo prebuild` to generate the native projects if you are using config plugins).
+
+The `expo run` commands will run a new build, install it in your emulated device, and launch you into your application.
+
+<Tabs tabs={["For iOS Simulator (MacOS Only)", "For Android Emulator"]}>
+
+<Tab >
 
 <InstallSection packageName="expo-dev-client" cmd={["expo run:ios"]} hideBareInstructions />
 
-or your Android emulator
+</Tab>
+
+<Tab >
 
 <InstallSection packageName="expo-dev-client" cmd={["expo run:android"]} hideBareInstructions />
 
-If you are eager to install your project on a physical device, we recommend using [EAS Build](eas-build.md) for the smoothest experience, but you can build and distribute the same as any other React Native application. Once it's installed, you're ready to start developing by running:
+</Tab>
+
+</Tabs>
+
+
+## Developing your application
+
+As you can see, creating a new native build from scratch takes long enough that you'll be tempted to switch tasks and lose your focus.
+
+Now that you have a custom client for you project installed on your device, though, you won't have to wait for the native build process again until you change the native runtime!
+
+Instead, you can start developing in a fraction of the time by running:
```

Comment:
```suggestion
Now all you need to do to start developing is to run:
```
This whole section feels just a little bit more sales-pitch-y than I would expect from a guide, but I also get that you're trying to communicate and emphasize the difference between native vs JS development. Maybe just changing this sentence is enough?